HabboHotel/Groups/Forums/GroupForumManager.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Dapper;
+using Plus.Database;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Groups.Forums
+{
+    public class GroupForumManager : IGroupForumManager
+    {
+        private readonly IDatabase _database;
+        readonly List<GroupForum> Forums;
+        private readonly IGroupManager _groupManager;
+
+        public GroupForumManager(IDatabase database, IGroupManager groupManager)
+        {
+            Forums = new List<GroupForum>();
+            _database = database;
+            _groupManager = groupManager;
+        }
+
+        public GroupForum? GetForum(int GroupId) => TryGetForum(GroupId, out GroupForum f) ? f : null;
+
+        public async Task<GroupForum> CreateGroupForum(Group group)
+        {
+            if (TryGetForum(group.Id, out GroupForum groupForum))
+                return groupForum;
+
+            using (var connection = _database.Connection())
+            {
+                await connection.ExecuteAsync("REPLACE INTO group_forums_settings (group_id) VALUES (@gp)", new { gp = group.Id });
```

Comment:
Try to assign clear names to variables including query parameters. I would use `@groupId` here.